src/AvTranscoder/properties/FileProperties.cpp

```diff
@@ -210,7 +210,7 @@ size_t FileProperties::getBitRate() const
     return _avFormatContext->bit_rate;
 }
 
-size_t FileProperties::getFileSize() const
+unsigned __int64 FileProperties::getFileSize() const
```

Comment:
is it possible to define more something like in TuttleOFX:
https://github.com/tuttleofx/TuttleOFX/blob/develop/libraries/tuttle/src/tuttle/common/system/compatibility.hpp

Reply:
Do you want me to define a new type in std namespace ?

e.g (this is pseudo-code)

```c++
// compatibility problems...
namespace std
{
#ifdef 32bit-OS // use https://stackoverflow.com/questions/1505582/determining-32-vs-64-bit-in-c to define the system type
	typedef Custom_size_t c_size_t // can hold 64-bits
#else 
	typedef ::size_t c_size_t
#endif
}
```

I'm not really use to c++ but doesn't it seem a bit weird ?

Reply:
LGTM

Reply:
Indeed  __uint64_t__ seems to be what you are looking for (it is provided by the C library).
Did you test it @adoussot ?

Reply:
Yes, i thought `unsigned __int64` was cross platform (which is not). so `uint64_t` is correct. Thanks!